datacube/scripts/product_extent.py

```diff
@@ -0,0 +1,73 @@
+import getopt
+import logging
+import sys
+from datetime import datetime
+
+import click
+from yaml import load
+
+from datacube.db_extent import ExtentUpload, parse_time
+from datacube.drivers.postgres import PostgresDb
+from datacube.index.index import Index
+from datacube.ui import click as ui
+from datacube.ui.click import cli
+
+_LOG = logging.getLogger('datacube-md-type')
+
+
+@cli.group(name='product_extent', help='Product Extents Tools')
```

Comment:
Can we just call it `extent`, having `_` in the subcommand feels odd, also we already have just `product` subcommand.

Reply:
Good idea, we're still working on making it feel integrated.